packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,137 @@
+
+```
+<nimble-accordion>
+    <nimble-accordion-item>
+        <span slot="heading">Panel one</span>
+    </nimble-accordion-item>
+    <nimble-accordion-item expanded>
+        <span slot="heading">Panel two</span>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <span slot="heading">Panel three</span>
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+Complex accordion group with components in slots and single expand-mode
+
+```
+<nimble-accordion expand-mode="single">
+    <nimble-accordion-item>
+        <nimble-checkbox slot="start"></nimble-checkbox>
+        <div slot="heading">Accordion one</div>
+        <nimble-select slot="end">
+            <nimble-option value="1">Option 1</nimble-option>
+            <nimble-option value="2">Option 2</nimble-option>
+            <nimble-option value="3">Option 3</nimble-option>
+        </nimble-select>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <nimble-checkbox slot="start"></nimble-checkbox>
+        <div slot="heading">Accordion two</div>
+        <nimble-number-field placeholder="number" slot="end"></nimble-number-field>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <nimble-checkbox slot="start"></nimble-checkbox>
+        <div slot="heading">Accordion three</div>
+        <nimble-select slot="end">
+            <nimble-option value="1">Option 1</nimble-option>
+            <nimble-option value="2">Option 2</nimble-option>
+            <nimble-option value="3">Option 3</nimble-option>
+        </nimble-select>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <nimble-checkbox slot="start"></nimble-checkbox>
+        <div slot="heading">Accordion four</div>
+        <nimble-number-field placeholder="number" slot="end"></nimble-number-field>
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+### API
+
+#### Nimble Accordion
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_
+    -   `appearance` - `block`, `outline`, `ghost`
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+The Figma design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` appearances used in other components. Because of the difference in border functionality and possibility of excess overridden css classes if patterns are used, shared styles will not be used, and new styles for the accordion will be created. The appearances of `block`, `outline`, and `ghost` will be implemented under the HTML attribute `appearance`, which will use conditional css styles based on `appearance`'s value (ex. :host([appearance='outline']) will have styling for the outline button).
```

Comment:
>shared styles will not be used

Could you clarify this? Are you saying that the accordion won't share styles with other components or that the accordion won't share styles across the three appearances?

Reply:
That's what I put in the spec, but I still can't decide if using the button styling patterns is worth it or not. The main conflict is that it's hard to know how much overriding we'll have to do if we use the button styling patterns, but at the same time it could save us from a lot of extra css since it mostly follows the same appearance states as the accordion. Maybe we can leave this as an open issue in the spec and I can first try using the button patterns to see if it works, and if not, switch to creating new styles for the accordion?

Reply:
I'm ok with leaving it unspecified in the spec.

My guess is that we shouldn't try too hard to share the styles with the buttons. We have several other components that support block and outline appearance (text field, text area, select) and I don't think any of them use the button styles (they do use the appearance behavior infrastructure). I'd suggest starting with the same pattern for the accordion.

Reply:
Sounds good, I included it in the open issues section and advised that we leave it open until we spend some time trying to implement it.